packages/ui/src/components/Table.svelte

```diff
@@ -0,0 +1,123 @@
+<!--
+// Copyright © 2021 Anticrm Platform Contributors.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import type { IntlString } from '@anticrm/status'
+
+  import { AnySvelteComponent } from '../types'
+  import Label from './Label.svelte'
+
+  interface Cell {
+    component: AnySvelteComponent
+    props?: Object
+  }
+
+  interface Column {
```

Comment:
Should we put this files to types.ts and reuse for type safety?

Reply:
@haiodo Probably I misunderstood you, we will get an error if try to provide mismatching columns definition, isn't that enough?
What I really would like to achieve is having parametric component to make sure we don't use non-existing doc fields, like:
```
export let data: T[]

interface ColumnProp<T> {
  property: string
  key?: keyof T
  value?: any
}
```